src/MPDevice.cpp

```diff
@@ -965,6 +965,11 @@ void MPDevice::startMemMgmtMode(bool wantData, bool wantFido,
 
 QByteArray MPDevice::getMemoryFirstNodeAddress(void)
 {
+    if (isBLE())
+    {
+        /* BLE: Address format is 2 bytes little endian. last 1 bit is node number (for pages that can accomodate 2 nodes) and first 15 bits are page address */
+        return QByteArray::fromHex("0200");
```

Comment:
I don't think this is correct.
Address format is page << 1
first address available (from the firmware code):
![image](https://user-images.githubusercontent.com/6170291/128888700-34275944-3aad-4902-b4f0-53c696087fe9.png)
so for the 1M / 2M / 32M version it is 0100